README.md

```diff
@@ -115,7 +113,19 @@ our company and culture evolve, so should this document.
 
 ### Invent
 
-#### Don't be afraid to push the boundaries of technology, but [be conservative with your tech stack](http://mcfunley.com/choose-boring-technology).
+#### New problems don't require new technology
+
+When building a software that solves a problem, it's unlikely that you need new technology to it.
+The things you are building should be pushing boundaries, not the things you are building them with.
+By default, you should [choose boring technology](http://mcfunley.com/choose-boring-technology).
+
+To make such technology decisions easy, it's important to maintain a set of best practices outlining what technology
+to use when solving particular classes of problems.
+
+Mobify best practices can be found [here](https://mobify.atlassian.net/wiki/display/PLAT/Coding+and+Design+Best+Practices).
```

Comment:
Yep - we try to be general when possible but this is also a tool we want Mobifyers use, and sometimes it's helpful to point to internal things. But when doing so, we should make it explicit that it's for Mobifyers only. (IMO anyways!)